.github/workflows/linux-ci.yml

```diff
@@ -76,37 +76,53 @@ jobs:
           cache: poetry
 
       - name: Install python deps
-        run: poetry install -E core -E backend --remove-untracked
+        run: |
+          set -eux
+          poetry install -E core -E backend
+          poetry run maturin develop
```

Comment:
with your approach, this step in not required since it's executed in the `build.py` at the `poetry install ...`